src/util/eventAttendee.js

```diff
@@ -27,8 +27,8 @@
  * @return {string} URI without a mailto prefix
  */
 export function removeMailtoPrefix(uri) {
-	if (uri.startsWith('mailto:')) {
-		return uri.substr(7)
+	if (uri.startsWith('mailto:') || uri.startsWith('MAILTO:')) {
```

Comment:
I missed this. is this change intentional?

Reply:
yeah? The ics I got from Patrick contains an uppercase `MAILTO:`. I figured `substring()` is case sensitive. Is that not the case?

Reply:
> Is that not the case?

Good question. Did you test that?

Reply:
uh, `startsWith` is definitely case sensitive